rfcs/90-re-support-initialization-via-config-string.md

```diff
@@ -0,0 +1,139 @@
+---
+author: xxchan <xxchan22f@gmail.com>
+status: draft
+updated_at: 2021-06-09
+deprecates:
+  - 13-remove-config-string.md
+---
+
+# GSP-90: Re-support Initialization Via Config String
```

Comment:
How about naming `Initialization Via Connection String`?

`connection string` looks widely used in computer science: https://en.wikipedia.org/wiki/Connection_string

Reply:
I don't remember why I name it `config string` 🤔

Reply:
Oh, it was the old name